UnameMeter.c

```diff
@@ -0,0 +1,57 @@
+/*
+htop - UnameMeter.c
+(C) 2004-2011 Hisham H. Muhammad
+Released under the GNU GPL, see the COPYING file
+in the source distribution for its full text.
+*/
+
+#include "UnameMeter.h"
+
+#include "CRT.h"
+
+#include <stdio.h>
+#include <string.h>
+#include <sys/utsname.h>
+
+/*{
+#include "Meter.h"
+}*/
+
+int UnameMeter_attributes[] = {
+   HOSTNAME
+};
+
+static void UnameMeter_updateValues(Meter* this, char* buffer, int size) {
+   (void) this;
+   struct utsname *data = NULL;
+   data = xMalloc(sizeof(struct utsname));
```

Comment:
is there a point in using malloc here? why not just:
`
struct utsname data;
uname(&data);
`